recipes/tokenizers/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
This shouldn't be required next version onwards since https://github.com/huggingface/tokenizers/pull/338 was merged into upstream.